### PR TITLE
Fix dependent packages showing up as changed in getAffectedPackages

### PR DESCRIPTION
**Pull request by jakebailey**
Fixes https://github.com/DefinitelyTyped/DefinitelyTyped/issues/67205

I think I probably incorrectly suggested the bad code in a change, not understanding what this code did. This just removes `...` which gives the right answer on DT:

```
dtslint-runner@0.0.188
Node version:  v20.9.0
Using local DefinitelyTyped at .
Running: git rev-parse --verify master
d0df83268470773861ff1bd2a6a9520b8c247d3d
Running: git diff master --name-status
M       types/node/v18/child_process.d.ts
M       types/node/v18/test/child_process.ts
Testing 1 changed packages: Set(1) { 'node/v18' }
Testing 9 dependent packages: Set(9) {
  'ssh2',
  'docker-modem',
  'dockerode',
  'docker-events',
  'node-ssh',
  'promise-sftp',
  'ssh2-sftp-client',
  'ssh2-sftp-client/v7',
  'tunnel-ssh'
}
```

### CODE REVIEW

File: packages/definitions-parser/src/get-affected-packages.ts [hunk 1]
```diff
@@ -52,6 +56,11 @@ export async function getAffectedPackages(
       )
     );
   }
+
+  console.log("changedPackageDirectories", changedPackageDirectories);
+  console.log("addedPackageDirectories", addedPackageDirectories);
+  console.log("allDependentDirectories", allDependentDirectories);
```
Comment (sandersn) [inline packages/definitions-parser/src/get-affected-packages.ts:62]: Don't we log similar derived information a little bit later? It's useful but probably overwhelming especially for large changes.

No recommendation, just an observation. More logging is better than none, so I guess leave it in.

**Review by sandersn — APPROVED** [review]

File: packages/definitions-parser/src/get-affected-packages.ts (hunk 1 above)
Comment (jakebailey) [reply packages/definitions-parser/src/get-affected-packages.ts]: forgot to push, this is not supposed to be there!

File: packages/definitions-parser/src/get-affected-packages.ts [hunk 2]
```diff
@@ -52,6 +56,7 @@ export async function getAffectedPackages(
       )
     );
   }
+
```
Comment (jakebailey) [inline packages/definitions-parser/src/get-affected-packages.ts:59]:
```suggestion
```

File: packages/definitions-parser/src/get-affected-packages.ts (hunk 1 above)
Comment (jakebailey) [reply packages/definitions-parser/src/get-affected-packages.ts]: (pushed now...)

**Comment by jakebailey** [comment]
Meta point, but would be nice to set up some sort of test DT that we can test these pnpm commands in or something.

**Comment by peterblazejewicz** [comment]
thanks!

**Pull request merged**